src/Io/MultipartParser.php

```diff
@@ -154,6 +175,14 @@ private function parsePost($headers, $body)
                     $matches[1],
                     $body
                 ));
+
+                if (strtoupper($matches[1]) === 'MAX_FILE_SIZE') {
+                    $this->maxFileSize = $body;
+
+                    if ($body == '0') {
+                        $this->maxFileSize = null;
+                    }
```

Comment:
Should probably cast to `int` above and use strict comparison here? This seems to be in line with how PHP parses this: https://github.com/php/php-src/blob/1c295d4a9ac78fcc2f77d6695987598bb7abcb83/main/rfc1867.c#L914

Reply:
SGTM :+1: